packages/css/package.json

```diff
@@ -18,6 +18,7 @@
     "clean": "rimraf dist/"
   },
   "dependencies": {
+    "@amsterdam/design-system-tokens": "workspace:*",
```

Comment:
What does this do?

Reply:
We import a design token directly in `paragraph.scss`, so it needs to be in the dependencies. However, we currently only export vanilla css files, not sass, so it should be in the `devDependencies` I think